contrib/systemd/README.md

```diff
@@ -5,6 +5,8 @@ author: Ben Boeckel
 date: 2015-03-22
 contributors: Abdo Roig-Maranges
 updated: 2015-03-25
+contributors: benutzer193
+updated: 2017-06-01
```

Comment:
I think those are good defaults and people who want to change them should be able to understand those options (as they are relatively self-explanatory) / find the options in systemd man files. What do you think?